### ISSUE TITLE
Completion ordering

### ISSUE THREAD

**Issue by ncannasse** [vshaxe/vshaxe#306]
Another case of (wrong) completion ordering, which lead me to TAB the wrong class:

![image](https://user-images.githubusercontent.com/1022912/49149345-21f92a00-f30a-11e8-90ac-69d19dc5469a.png)

**Comment by Gama11** [comment]
That's definitely counter-intuitive sorting, but it's mostly up to VSCode I think.

**Comment by ncannasse** [comment]
Is it?

**Comment by Gama11** [comment]
Yep, at least in this case. I think for other completion items, we set an explicit `sortText`, but that led to a lot of other issues with types / packages. I opened a VSCode issue: https://github.com/Microsoft/vscode/issues/67887

**Comment by ncannasse** [comment]
Another order related issues:

![image](https://user-images.githubusercontent.com/1022912/67944846-c4d2bc00-fbdd-11e9-8130-97ceefc6dc28.png)

We should get `init` before `initVisual` here. (tested with Suggestions:first mode in VSCode)

**Comment by Gama11** [comment]
That's a much more straightforward case, we don't do anything tricky with `sortText` or the label for fields. Maybe open a VSCode issue about that one if you want to see it changed, we can't really do much about it.
